### PR TITLE
Port legacy tests to Swift

### PR DESCRIPTION
**Pull request by ricardopereira**
#627

### CODE REVIEW

**Comment by ricardopereira** [comment]
BTW, why is `RSC14a` missing from the [v1.0 docs](https://docs.ably.io/client-lib-development-guide/features/)?

**Comment by mattheworiordan** [comment]
> BTW, why is RSC14a missing from the v1.0 docs?

@ricardopereira I think because it's covered elsewhere, i.e. https://docs.ably.io/client-lib-development-guide/features/#RSA2 and https://docs.ably.io/client-lib-development-guide/features/#RSA11

File: Ably.xcodeproj/xcshareddata/xcschemes/Ably.xcscheme [hunk 1]
```diff
@@ -20,6 +20,20 @@
                ReferencedContainer = "container:Ably.xcodeproj">
             </BuildableReference>
          </BuildActionEntry>
+         <BuildActionEntry
```
Comment (tcard) [inline Ably.xcodeproj/xcshareddata/xcschemes/Ably.xcscheme]: We might as well just delete them, no?

File: Spec/RealtimeClientPresence.swift [hunk 2]
```diff
@@ -3814,6 +3844,47 @@ class RealtimeClientPresence: QuickSpec {
...
+                        let partialDone = AblyTests.splitDone(4, done: done)
+                        channel.presence.enterClient(john, data: nil) { error in
+                            expect(error).to(beNil())
+                            partialDone()
+                        }
+                        channel.presence.enterClient(max, data: nil) { error in
+                            expect(error).to(beNil())
+                            partialDone()
+                        }
+                        channel.presence.subscribe { message in
```
Comment (tcard) [inline Spec/RealtimeClientPresence.swift]: Shouldn't this subscription be done before the `enterClient`s, so that we make sure we don't miss them?

**Comment by mattheworiordan** [comment]
@tcard @ricardopereira I was hoping porting these tests would result in a green build.  What's the issue that is preventing a green build then?

**Pull request merged**